### ISSUE TITLE
How can I use trained model to evaluate the performance on tianchi dataset？

### ISSUE THREAD

**Issue by kasteric** [qcf-568/Det_Baseline#6]
Hi, Author,

I am wondering after 2 epochs of training on tianchi training dataset ( the training process was often killed for no reason, so I resumed automatically after it got killed, not knowing whether this led to the right training process), how could I use the trained checkpoint to evaluate on the validation set? Now I already have a folder called "test_set_A_rename" folder, what should I do next? Should I convert this folder into a pickle file first? Thanks a lot for your generous help and unwavering support!

**Comment by qcf-568** [comment]
Hello,
1. What is the code that you used for training? Is the Baseline.zip that I have provided? You can attach the bug report and I will help to fix it.

2. If you want to evaluate the performance score on a test dataset, you should have the ground-truth of it. If you just want to get the predictions on a dataset, you can infer the model with the [tta_infer.py](https://github.com/qcf-568/Det_Baseline/blob/main/tta_infer.py). Please modify the "test_img_dir" on Line68 and modify Line77 to your custom result saving method.

Thank you~
